README.md

```diff
@@ -23,40 +23,119 @@ $ npx @commercetools-frontend/create-mc-app my-new-custom-application-project --
 
 ## Developing application-kit packages
 
-Install the dependencies (uses yarn workspaces):
+### Initial Installation
```

Comment:
This all section can be removed to be honest. We can mention to check and ensure to have the `.env.local` file but I wouldn't document its content. Instead you can add more information and give hints in the template file itself.

Reply:
![Screen Shot 2021-09-21 at 2 44 09 PM](https://user-images.githubusercontent.com/13600903/134229337-744bff5c-5ed5-465e-83f2-548db136f911.png)

@emmenko 
When you initially pull down the repo this is the result when you run yarn build:watch after adding the environment variables without running yarn build first.